controls/raddiagram/features/global-settings.md

```diff
@@ -12,34 +12,49 @@ position: 15
+* __MaximumZoom__: Gets or sets the maximum zoom level. The default value of this constant is 5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
 
-* __RotationAngleSnap__ - Gets or sets the snapping angle when rotating items. The default value of this constant is 5 degrees. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
+* __MinimumZoom__: Gets or sets the minimum zoom level. The default value of this constant is 0.5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
 
-* __RotationHitTestRadius__ - gets or sets the radius around the rotation thumb where a click operation activates the thumb. The default value of this constant is 7px. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
+* __RotationAngleSnap__: Gets or sets the snapping angle when rotating items. The default value of this constant is 5 degrees. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
 
-* __RoutingGridSize__ - gets or sets the routing grid cell size. The value of this constant is used in the layout algorithm used to layout the __RadDiagramConnections__. The default value of this constant is 40px.			
+* __RotationHitTestRadius__: Gets or sets the radius around the rotation thumb where a click operation activates the thumb. The default value of this constant is 7px. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
 
-* __SelectionHitTestRadius__ - The radius of the area around each item which is considered selectable. The default value of this constant is 10px.			
+* __RoutingGridSize__: Gets or sets the routing grid cell size. The value of this constant is used in the layout algorithm used to layout the __RadDiagramConnections__. The default value of this constant is 40px.			
 
-* __StartDragDelta__ - The minimum distance that a mouse move operation has to cover to start a drag action. The default value of this constant is 4px.			
+* __SelectionHitTestRadius__: Gets or sets the radius of the area around each item which is considered selectable. The default value of this constant is 10px.			
+
+* __StartDragDelta__: Gets or sets the minimum distance that a mouse move operation has to cover to start a drag action. The default value of this constant is 4px.		
+
+* __MinimumAdornerSize__: Gets or sets a __double__ value that determines the minimum size of the diagram items adorner.
+
+* __MinimumShapeSize__: Gets or sets a __double__ value that determines the minimum size of the diagram shapes.
+
+> We recommend to use matching values for the __MinimumShapeSize__ and __MinimumAdornerSize__ properties. Otherwise, the adorner won't match the shape's size when reaches its minimum size.

```

Comment:
"... when its minimum value is reached" or something similar seems a little bit better IMO.